Attachements always exceed Size Limit

<!--
Please fill out the following template to make solving your problem easier and faster for us.
This is only a guideline. If you think that parts are unneccessary for your issue, feel free to remove them.

Remember to hide/obfuscate personal and confidential information, 
such as names, global IP/DNS adresses and especially passwords, if neccessary.
-->

### Subject of the issue
<!-- Describe your issue here.-->
Attachements wont be uploaded with Error "Attachment size limit exceeded with this file: SizeLimit" even with SizeLimit beeing set above the file SIze.

### Your environment
<!-- The version number, obtained from the logs or the admin page -->
* Bitwarden_rs version: 1.14.1-91dd1947
<!-- How the server was installed: Docker image / package / built from source -->
* Install method: Docker-Image
* Clients used: Web Vault, Windows Client
* Reverse proxy and version: <!-- if applicable -->
* Version of mysql/postgresql: mariadb 10.4
* Other relevant information: 

### Steps to reproduce
<!-- Tell us how to reproduce this issue. What parameters did you set (differently from the defaults)
and how did you start bitwarden_rs? -->
Set per User attachement Limit to 1000KB. Tried it in the admin page as well as in the docker-compose. Tried to upload files way below this limit, which throws the error above.


### Expected behaviour
<!-- Tell us what should happen -->
The attachement should be abloaded.

### Actual behaviour
<!-- Tell us what happens instead -->
Uploading the attachement gets rejected by the error "Attachment size limit exceeded with this file: SizeLimit".

### Relevant logs
<!-- Share some logfiles, screenshots or output of relevant programs with us. -->
[2020-03-30 12:52:36][multipart::server][INFO] Multipart::with_boundary(_, "----WebKitFormBoundaryzCinABGOv5R6D0d0")
[2020-03-30 12:52:36][error][ERROR] Attachment size limit exceeded with this file: SizeLimit
[2020-03-30 12:52:36][response][INFO] POST /api/ciphers/<uuid>/attachment multipart/form-data (post_attachment) => 400 Bad Request

--- a/src/api/core/ciphers.rs
+++ b/src/api/core/ciphers.rs
@@ -665,8 +665,8 @@ fn post_attachment(
     let size_limit = if let Some(ref user_uuid) = cipher.user_uuid {
         match CONFIG.user_attachment_limit() {
             Some(0) => err_discard!("Attachments are disabled", data),
-            Some(limit) => {
-                let left = limit - Attachment::size_by_user(user_uuid, &conn);
+            Some(limit_kb) => {
+                let left = (limit_kb * 1024) - Attachment::size_by_user(user_uuid, &conn);
                 if left <= 0 {
                     err_discard!("Attachment size limit reached! Delete some files to open space", data)
                 }
@@ -677,8 +677,8 @@ fn post_attachment(
     } else if let Some(ref org_uuid) = cipher.organization_uuid {
         match CONFIG.org_attachment_limit() {
             Some(0) => err_discard!("Attachments are disabled", data),
-            Some(limit) => {
-                let left = limit - Attachment::size_by_org(org_uuid, &conn);
+            Some(limit_kb) => {
+                let left = (limit_kb * 1024) - Attachment::size_by_org(org_uuid, &conn);
                 if left <= 0 {
                     err_discard!("Attachment size limit reached! Delete some files to open space", data)
                 }

